link/src/main/java/com/stripe/android/link/LinkActivity.kt

```diff
@@ -45,46 +74,131 @@ internal class LinkActivity : ComponentActivity() {
         }
 
         setContent {
+            val vm = viewModel ?: return@setContent
```

Comment:
Nit: Can the view model not be created here?

Reply:
I moved it out of setContent